node/cmd/guardiand/node.go

```diff
@@ -534,6 +543,15 @@ func runNode(cmd *cobra.Command, args []string) {
 		logger.Fatal("Please specify --terraContract")
 	}
 	if *testnetMode || *unsafeDevMode {
+		if *terra2WS == "" {
+			logger.Fatal("Please specify --terra2WS")
+		}
+		if *terra2LCD == "" {
+			logger.Fatal("Please specify --terra2LCD")
+		}
+		if *terra2Contract == "" {
+			logger.Fatal("Please specify --terra2Contract")
+		}
```

Comment:
This says we must run terra2 in testnet, so as soon as this gets merged the next testnet deploy must have terra2. Is that what we want?

Reply:
Yes I think so. The terra 2 node was setup today and should be done being configured tomorrow in theory.